content/en/docs/gke/deploy/project-setup.md

```diff
@@ -63,9 +63,4 @@ cluster for you.
   Cloud IAP is recommended for production deployments or deployments with access 
   to sensitive data. You can skip this step if you want to test Kubeflow
   in a non-production environment.
-* Choose one of the following ways to deploy Kubeflow:
-
-  * [Using the UI](/docs/gke/deploy/deploy-ui). This option provides a simple
-    way to deploy Kubeflow.
-  * [Using the CLI](/docs/gke/deploy/deploy-cli). This option provides more
-    control over the deployment process.
+* Follow the [instructions](/docs/gke/deploy/deploy-cli) to deploy Kubeflow
```

Comment:
Suggestion: 

- Use a blank line to separate the paragraphs
- Add an emphasis on on the CLI method for deployment

```markdown

* Follow the [instructions](/docs/gke/deploy/deploy-cli) to deploy Kubeflow using the CLI
```

Reply:
(Though using the UI is no longer supported from v1.1, stating that it's a CLI method would be useful IMHO, in case someone didn't see this relatively significant change from 1.0 to 1.1.)

Reply:
Thanks. Done.